### ISSUE TITLE
Дата оплаты по бизнес-карте ("Зарезервировано") в выписке по счёту

### ISSUE THREAD

**Issue by tzurbaev** [TinkoffCreditSystems/business-openapi#13]
В приложении Tinkoff Business у операций по бизнес-картам есть две даты: Зарезервировано и Проведено. API выписки по счёту (https://business.tinkoff.ru/openapi/docs#operation/getApiV1Bank-statement), если я правильно понял, во всех полях с датой отдаёт только дату проведения платежа.

Есть ли возможность добавить дату "Зарезервировано" в ответ этого метода API? Используется сценарий Селф-Сервис.

**Comment by ttemnov** [comment]
Вы имеете в виду когда был создан платеж? Мы в Q1-Q2 планируем переделать выписки и учтем этот вопрос.

**Comment by tzurbaev** [comment]
Да, дата создания платежа. Скажем, сегодня я оплатил бизнес-картой покупку в магазине, а завтра-послезавтра платёж будет проведён банком. Пока платёж не проведён, отображается дата покупки. После проведения даты покупки уже нет. Вот её и хотелось бы иметь рядом с датой проведения.

**Comment by ttemnov** [comment]
Да, обязательно это учтем :)
